rdr_service/dao/study_nph_dao.py

```diff
@@ -474,6 +500,63 @@ def __init__(self):
     def get_id(self, obj: OrderedSample):
         return obj.id
 
+    @staticmethod
+    def _is_order_cancelled(order: Order) -> bool:
+        return order.status == "cancelled"
+
+    @staticmethod
+    def _is_ordered_sample_cancelled(ordered_sample: OrderedSample) -> bool:
+        return ordered_sample.status == "cancelled"
+
+    def get_biospecimens_for_order(self, nph_participant: Participant, order: Order) -> Iterator[Dict[str, Any]]:
+        nph_stored_sample_session = NphStoredSampleDao()
+        nph_study_category_dao = NphStudyCategoryDao()
+        with self.session() as session:
+            ordered_samples_for_participant: Iterable[OrderedSample] = list(
+                session.query(OrderedSample).filter(OrderedSample.order_id == order.id).all()
+            )
+            for ordered_sample in ordered_samples_for_participant:
+                parent_study_category = nph_study_category_dao.get_parent_study_category(order.category_id)
+                nph_module_id = nph_study_category_dao.get_parent_study_category(parent_study_category.id)
+                sample_processing_ts = ordered_sample.collected if not ordered_sample.parent is None else None
+                collectionDateUTC = _format_timestamp((ordered_sample.parent or ordered_sample).collected)
+                processingDateUTC = _format_timestamp(sample_processing_ts)
+                finalizedDateUTC = _format_timestamp(ordered_sample.finalized) if ordered_sample.finalized else None
+                sample_is_cancelled = (
```

Comment:
Can we use snake_case instead of camelCase for the actual variable names to follow suite for the rest of the declarations.

Reply:
Fixed other variable names to match snake case convention